src/vanilla/store.ts

```diff
@@ -795,7 +795,7 @@ export const createStore = () => {
   }
 }
 
-type Store = ReturnType<typeof createStore>
+export type Store = ReturnType<typeof createStore>
```

Comment:
While it's controversial, this is intentional not to export the type here. We basically strictly control what type to export.

Reply:
@dai-shi Thanks for the quick feedback 🤗 I have reverted some of the work.